### PR TITLE
MOTECH-2895: Migrate PR jobs to Travis CI

### PR DESCRIPTION
**Pull request by mkruszynski**

### CODE REVIEW

**Comment by sebbrudzinski** [comment]
test this please

File: pom.xml [hunk 1]
```diff
@@ -1094,6 +1094,23 @@
                                     </resources>
                                 </configuration>
                             </execution>
+                            <execution>
```
Comment (sebbrudzinski) [inline pom.xml:1097]: Can't this be a part of the same execution step?

Comment (mkruszynski) [reply pom.xml:1097]: No, every execution markup is assigned to one directory, where the copied files are placed.

**Comment by sebbrudzinski** [comment]
@mkruszynski Let's only remove log4j.xmls from tests. This should be sufficient for our 4MB limit.

**Comment by sebbrudzinski** [comment]
test this please

**Comment by mkruszynski** [comment]
test this please

**Pull request merged**